### ISSUE TITLE
json decoding issue

### ISSUE THREAD

**Issue by HackedK** [Tim-Schwalbe/python_mattermost#3]
Hi, I am trying to use your script, and when I run it, it returns the error: "No JSON object could be decoded". My input is: ./send_file.py <http://mattermostserver.com> <access_token> <channel_id> <file_path/sample_file.txt>

environment: CENTOS7, python 2.7.5

Thanks

**Comment by Tim-Schwalbe** [comment]
This is working for me, with this cmd:
./send_file.py https://mattermost.mycompany.de "$MATTERMOST_ACCESS_TOKEN" "$CHANNEL_ID" "./screenshot.jpg" -am "Hey" -c "#FF0000"'

**Comment by HackedK** [comment]
The problem i had was that i did not have enough argument thanks. Now it returns {'attachment': [{'color': '#FFFFFF', 'text':' testing}]} but I do not see the file in the channel.

**Comment by Tim-Schwalbe** [comment]
did you use the correct channel id? it's not the name

**Comment by HackedK** [comment]
yes im pretty sure im using correct channel ID. For url, do I need to prompt the entire URL like https://mattermost/testing/channels/town-square

**Comment by Tim-Schwalbe** [comment]
You are using the name. The id is something like this
ID: a4af4dk9itruuj1fdaokjidpr4h

**Comment by HackedK** [comment]
right i am using channel ID which looks like the one you described. The url is the one I copied from the web, so in command would be different, in channel, I have channel ID, not the name.

**Comment by Tim-Schwalbe** [comment]
are you using " to separate the parameters?

**Comment by HackedK** [comment]
am I wrap each parameters with "? no, I didnt do it, and I just tried it, wrapping each parameter inside double quotes, and still not uploading a text file....
fyi, I am not sure if I also need an activated bot or incoming webhook for this.....
